docs/lib/storage/fragments/flutter/list.md

```diff
@@ -0,0 +1,27 @@
+You can list all of the objects uploaded under a given prefix. This will list all public files:
+
+```dart
+try {
+  ListResult res = await Amplify.Storage.list(path: "/");
```

Comment:
You can specify path: "" or `null` to list all public files. Android documentation seems wrong.

Reply:
Done.